updates/update_1_10_11_.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/bash
```

Comment:
These files should only contain the part after `## BACKWARD FIXES ( for older images )`.

The updates folder should only contain one file initially with the contents of the current `update.sh` after `## BACKWARD FIXES ( for older images )`.

Reply:
I created one script - the contents of *BACKWARD FIXES* part and named it 1.10.11.sh (the version before latest cleanup of update.sh). Feel free to fill the updates folder, just keep in mind that my mechanism, as it is written right now, works with files of the following format: x.y.z.sh where x.y.z is the version that the system will update to.